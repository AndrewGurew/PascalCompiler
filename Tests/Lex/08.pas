a:=1
b+=2
c-=3
g*=5.6e3
f/=5.1
if(a<>b) being
end.
