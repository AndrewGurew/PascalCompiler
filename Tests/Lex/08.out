Position            Type                     Text                Value               
-------------------------------------------------------------------------------------
(1,1)               Id                       a                   a                   
(1,2)               Assign                   :=                  :=                  
(1,4)               Integer                  1                   1                   
(2,2)               Id                       b                   b                   
(2,3)               Plus and assign          +=                  +=                  
(2,5)               Integer                  2                   2                   
(3,2)               Id                       c                   c                   
(3,3)               Minus and assign         -=                  -=                  
(3,5)               Integer                  3                   3                   
(4,2)               Id                       g                   g                   
(4,3)               Mult and assign          *=                  *=                  
(4,5)               Double                   5.6e3               5600.0              
(5,2)               Id                       f                   f                   
(5,3)               Div and assign           /=                  /=                  
(5,5)               Double                   5.1                 5.1                 
(6,2)               If                       if                  if                  
(6,4)               Left bracket             (                   (                   
(6,5)               Id                       a                   a                   
(6,6)               Not equal                <>                  <>                  
(6,8)               Id                       b                   b                   
(6,9)               Right bracket            )                   )                   
(6,11)              Id                       being               being               
(7,2)               End                      end                 end                 
(7,5)               Dot                      .                   .                   
(8,5)               End of file              EndOfFile           EndOfFile           
